lib/dbutils.js

```diff
@@ -13,6 +13,21 @@ dbu.conversions = {
         read: JSON.parse,
         type: 'blob'
     },
+    string: {
+        write: function(value) {
+            if (value) {
```

Comment:
Also note here that for `value = 0` you'll still get a number out, not a string.